lib/daru/io/io.rb

```diff
@@ -90,6 +90,12 @@ def from_csv path, opts={}
         Daru::DataFrame.new(hsh,daru_options)
       end
 
+      def from_html path
+        page = Mechanize.new.get(path)
+        list = from_html_get_list(page)
+        from_html_get_df_list(list)
```

Comment:
Yes, splitting methods is a good thing. The problem is how to select right splitting point, and right names for methods. Typically, an approach "write long method, than divide it in 3" is not the most effective.